diff --git a/node_modules/@types/prompts/index.d.ts b/node_modules/@types/prompts/index.d.ts
index e3124b0..f4fac10 100644
--- a/node_modules/@types/prompts/index.d.ts
+++ b/node_modules/@types/prompts/index.d.ts
@@ -91,6 +91,7 @@ declare namespace prompts {
         mask?: string | PrevCaller<T, string | Falsy> | undefined;
         stdout?: Writable | undefined;
         stdin?: Readable | undefined;
+        submitOnEscapeKey?: boolean | undefined;
     }
 
     type Answers<T extends string> = { [id in T]: any };
