diff --git a/node_modules/prompts/lib/elements/prompt.js b/node_modules/prompts/lib/elements/prompt.js
index b793330..836de28 100644
--- a/node_modules/prompts/lib/elements/prompt.js
+++ b/node_modules/prompts/lib/elements/prompt.js
@@ -26,6 +26,9 @@ class Prompt extends EventEmitter {
     const isSelect = [ 'SelectPrompt', 'MultiselectPrompt' ].indexOf(this.constructor.name) > -1;
     const keypress = (str, key) => {
       let a = action(key, isSelect);
+      if (key.name === 'escape' && a === 'exit' && opts.submitOnEscapeKey) {
+        a = 'submit'
+      }
       if (a === false) {
         this._ && this._(str, key);
       } else if (typeof this[a] === 'function') {
